.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
+  "recommendations": [
+    "esbenp.prettier-vscode",
+    "denoland.vscode-deno"
```

Comment:
```suggestion
    "esbenp.prettier-vscode",
    "denoland.vscode-deno",
    "dbaeumer.vscode-eslint"
```

Reply:
Hmm, for some reason I remember that "format on save" for eslint extension caused me some headaches before (made some unnecessary git noise), but we run eslint in CI anyway, so I think it should be fine.

Reply:
Added eslint extension to the list.

Reply:
i *think* that's due to both trying to format on save?

i think this is what solved it for me, i have a few eslint related things in my settings

```json
{
	"editor.defaultFormatter": "esbenp.prettier-vscode",
	"editor.codeActionsOnSave": { "source.fixAll.eslint": true }
}